MiniEngine/Core/Core_VS14.vcxproj

```diff
@@ -406,13 +406,13 @@
     </ClCompile>
   </ItemDefinitionGroup>
   <ImportGroup Label="ExtensionTargets">
-    <Import Project="..\Packages\WinPixEventRuntime.1.0.170625002\build\WinPixEventRuntime.targets" Condition="Exists('..\Packages\WinPixEventRuntime.1.0.170625002\build\WinPixEventRuntime.targets')" />
+    <Import Project="..\Packages\WinPixEventRuntime.1.0.170918004\build\WinPixEventRuntime.targets" Condition="Exists('..\Packages\WinPixEventRuntime.1.0.170918004\build\WinPixEventRuntime.targets')" />
```

Comment:
Should MiniEngine vcxproj and config files be left as is in develop-dxr?

Reply:
I'd take whatever master is doing, generally shouldn't be doing anything to the miniengine core from develop-dxr